### ISSUE TITLE
Is there any pre-trained model for SUM-GAN-AED?

### ISSUE THREAD

**Issue by DucNT2002** [Aria-Minaidi/Self-Attention-based-GANs-for-Video-Summarization#1]
I've tried cloning the code and training the SUM-GAN-AED model exactly as described in the readme.md file, however, the results are not as expected (F-score is only around 40+) and the loss are really unstable. Therefore, I would like to ask if there are any pre-trained weights (for the AED model) that are publicly available? I would be very grateful if there are.
